R/susie_ash.R

```diff
@@ -195,7 +195,6 @@
 #'
 #' @importFrom stats var
 #' @importFrom utils modifyList
-#' @importFrom mr.ash.alpha mr.ash coef.mr.ash
```

Comment:
@danielnachun let's remove this file from pecotmr. We have a separate package for it now and eventually this is going into susieR

Reply:
I'll remove this in my next commit.